### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by SimonGideon**
In this Pull Request, The following was implemented: 
:heavy_check_mark: Name field accepts up to 30 chars.
:heavy_check_mark: Email validation.
:heavy_check_mark: Message field accepts up to 500 chars.
:heavy_check_mark: Placeholder in every filed text.
:heavy_check_mark: A user cannot submit the form if one or more of the fields is empty.
:heavy_check_mark: Collect data by using the [Formspree service](https://formspree.io/html).

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @SimonGideon 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been done individually and this is a pair programming project. Everyone on your team should make a commit in this PR.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Index.html [hunk 1]
```diff
@@ -224,5 +224,32 @@ <h1>I’m Simon
...
+                <span>Contact me</span>
+                <p>If you have an application you are interested
+                    in developing, a feature that you need built or a project that needs coding. I'd love to help with it
+                </p>
+            </div>
+            <div class="contact-form">
+                <form action="https://formspree.io/f/moqrolqj" method="post">
+                    <ul>
+                        <li>
+                            <input type="text" id="name" name="name" placeholder="user name" max="30" required>
```
Comment (vickymarz) [inline Index.html:240]: - [ ] Kindly take note that the right attribute is `maxlength` and not `max`. Please fix it 👍🏽

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @SimonGideon, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 1 above)
Comment (SimonGideon) [reply Index.html:240]: Thanks :smiley: (**Max** - Specifies the maximum value for an <input> element) I had to read through this again to get the difference.
(**Max-length** - Specifies the maximum number of characters allowed in an <input> element) 🙌
*NOTED* :100:

**Pull request merged**